client/guidestours/index.js

```diff
@@ -183,22 +290,32 @@ export default class GuidesTours extends Component {
 			return null;
 		}
 
-		const { bullseye, dialog } = this.getStepPositions();
-		const stepCoords = posToCss( dialog );
-		const pointerCoords = posToCss( bullseye );
+		let StepComponent = GuidesStep;
+		switch ( this.state.currentStep.type ) {
+			case 'GuidesFirstStep':
```

Comment:
I actually nuked this bit altogether in favor of a map lookup, which IMO delays the need for something more sophisticated in the config file.